DEVX_ROADMAP.md

```diff
@@ -1,51 +1,44 @@
-  * [crypto.move](https://github.com/MystenLabs/sui/blob/4ed29873e35cad9d709dbd9370e17d261446e873/crates/sui-framework/sources/crypto/elliptic_curve.move): This contains various utility methods for on-chain signature verification. It also provides a method to recover Secp256k1 signature to its public key, bulletproof verification, and Pedersen commitments. See this work in action in the Cryptography (math) [example](https://github.com/MystenLabs/sui/tree/main/sui_programmability/examples/math).
-  * [elliptic_curve.move](https://github.com/MystenLabs/sui/tree/main/sui_programmability/examples/math): This introduces a RistrettoPoint struct, which represents an elliptic curve point on the Ristretto255 sub-group of curve25519. Currently, we support the addition and subtraction of elements in the Ristretto255 sub-group, and operations of scalars within the field of curve25519.
-  * HKDF, AES, ECDH
+
+* Intent signing support which will include an intent struct to be serialized and signed in addition to the transaction data [[Issues](https://github.com/MystenLabs/fastcrypto/issues/26)].
+* [Rust SDK] Removing reliance on String and Json Values in Rust Transaction Builder, making it more Rust friendly
+* Adding denomination conversion functionality, which converts SUI to MIST.
+* [TypeScript] Adding support for deserializing transactions.
+
+## Sui Move
+
+* Crypto.move will be split into its individual modules organized by crypto primitives [[PR](https://github.com/MystenLabs/sui/pull/4653)].
+* Supporting dynamic child access requires additional time due to newer complexities discovered and we are actively working on enabling it on devNet ASAP. 
+* Support for passing mutable coins instead of owned.
+* New autogenerated docs for the Sui Framework [[link](https://github.com/MystenLabs/sui/tree/main/crates/sui-framework/docs)].
+* Enable shared coins via Safe module [[PR](https://github.com/MystenLabs/sui/pull/4680)].
+* Support for type reflection [[PR](https://github.com/move-language/move/pull/566)].
+* Support for `_`'s in integer literals (e.g., `let one_million = 1_000_000`).
 
 ## Sui CLI
-* Ability to use Sui CLI on full nodes ([issue](https://github.com/MystenLabs/sui/pull/4404))
-* Generate keypair now requires signature scheme flag
-  * `sui keytool generate ed25519` or `sui keytool generate secp256k1`
-  * `sui client new-address ed25519` or  `sui client new-address secp256k1`
-* Key derivation for Ed25519 following SLIP-0010
+* Intent signing support: Same as in SDK. Also, making the Sui keystore ChainID aware as well as password protected.
+* Source verification for dependent packages enforced at publish time.
+
+## Faucet
+* As a precursor to integrating Faucet into Sui CLI/SDKs/Wallet, rate-limiting will be enforced on the faucet server to 5 requests per minute and no more than 20 requests in 24 hours, originating from the same ip address.
```

Comment:
```suggestion
* As a precursor to integrating Faucet into Sui CLI/SDKs/Wallet, rate-limiting will be enforced on the faucet server to no more than 20 requests in 24 hours, originating from the same ip address.
```

Based on[ internal discussion](https://mysten-labs.slack.com/archives/C03GKUEA5PF/p1665009940561759?thread_ts=1665008605.936689&cid=C03GKUEA5PF), we will just keep the latter for now